packages/dd-trace/src/guardrails/index.js

```diff
@@ -57,6 +57,9 @@ function guard (fn) {
     var result = fn()
     telemetry('complete', ['injection_forced:' + (forced && initBailout ? 'true' : 'false')])
     log.info('Application instrumentation bootstrapping complete')
+    // Ensure the instrumentation source is set for the current process and potential 
+    // child processes.
+    process.env.DD_INSTRUMENTATION_SOURCE = 'ssi'
```

Comment:
Guardrails should likely not be touched. They just guarantee that nothing breaks with auto instrumentation.